stackit/internal/services/ske/cluster/resource.go

```diff
@@ -491,6 +502,22 @@ func (r *clusterResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 					},
 				},
 			},
+			"network": schema.SingleNestedAttribute{
+				Description: "Network block as defined below.",
+				Optional:    true,
+				Attributes: map[string]schema.Attribute{
+					"id": schema.StringAttribute{
+						Description: "ID of the STACKIT Network Area (SNA) network into which the cluster will be deployed.",
+						Optional:    true,
+						Validators: []validator.String{
+							validate.UUID(),
+						},
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.RequiresReplace(),
```

Comment:
why does it require replace? Did you test it or was it confirmed with the team?

Reply:
Confirmed with the team that it can never be changed after creation